build_image/docker/agent/ansible/Dockerfile.in

```diff
@@ -2,7 +2,7 @@
 #
 # @see https://github.com/hyperledger/cello/blob/master/docs/worker_ansible_howto.md
 #
-FROM _DOCKER_BASE_
+FROM python:3.6
 MAINTAINER Tong Li <litong01@us.ibm.com>
```

Comment:
Better use the following information?

MAINTAINER Hyperledger Cello Team <github.com/hyperledger/cello>

Reply:
I see this is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) - perhaps fix this in a following PR?